h/models/user_deletion.py

```diff
@@ -0,0 +1,37 @@
+from datetime import datetime
+
+from sqlalchemy import func
+from sqlalchemy.orm import Mapped, mapped_column
+
+from h.db import Base
+from h.models import helpers
+
+
+class UserDeletion(Base):
+    """A record of a user account that was deleted."""
+
+    __tablename__ = "user_deletion"
+
+    id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
+    userid: Mapped[str]
+    requested_at: Mapped[datetime] = mapped_column(
+        default=func.now(),  # pylint:disable=not-callable
+    )
+    requested_by: Mapped[str]
+    tag: Mapped[str]
+    created_at: Mapped[datetime]
+    num_annotations: Mapped[int]
```

Comment:
Might as well use the modern SQLAlchemy way?

Note: this makes all these columns not-nullable.